index.js

```diff
@@ -25,15 +25,16 @@ export const pageIndicator = {
 };
 
 export const ribbon = {
-  base: 'py-4 px-8 border rounded-4 inline-flex last-child:mb-0',
-  info: 'border-aqua-50 bg-aqua-50',
-  success: 'border-green-100 bg-green-100',
-  warning: 'border-yellow-100 bg-yellow-100',
-  error: 'border-red-100 bg-red-100',
-  disabled: 'border-bluegray-300 bg-bluegray-300',
-  sponsored: 'border-aqua-200 bg-aqua-200',
-  neutral: 'border-gray-100 bg-gray-100',
-  outlined: 'border-bluegray-300 bg-white',
+  base: 'py-4 px-8 border rounded-4 inline-flex last:mb-0',
+  info: ' i-bg-$color-badge-info-background i-text-$color-badge-info-text',
+  success: ' i-bg-$color-badge-positive-background i-text-$color-badge-positive-text',
+  warning: ' i-bg-$color-badge-warning-background i-text-$color-badge-warning-text',
+  error: ' i-bg-$color-badge-negative-background i-text-$color-badge-negative-text',
+  disabled: 'i-border-$color-badge-disabled-background i-bg-$color-badge-disabled-background i-text-$color-badge-disabled-text',
+  sponsored: 'i-border-$color-badge-price-background i-bg-$color-badge-price-background i-text-$color-badge-price-text',
+  neutral: 'i-border-$color-badge-neutral-background i-bg-$color-badge-neutral-background i-text-$color-badge-neutral-text',
+  roundedTopRightBottomLeft: 'rounded-tr-0 rounded-bl-0',
+  roundedTopLeftBottomRight: 'rounded-tl-0 rounded-br-0',
```

Comment:
I've seen usages of these classes together with ribbon so I guess it is good those are added here 👍  but I'm just curious if I missed a conversation about us deciding to add those here? Do we want our consumers to clean up a little bit or what is the purpose? 🙇‍♀️

Reply:
no there was no discussion on adding them. I simply just moved them as they were used in `Tag`
<img width="512" alt="Screenshot 2023-06-09 at 09 12 58" src="https://github.com/warp-ds/component-classes/assets/6739459/90e380ab-2165-4235-b6e6-08c74d650586">

Reply:
Alright, didn't now about `Tag`. Are we adding it to the docs?

Reply:
Couldn't find any usages of it in finn at least